### ISSUE TITLE
Does anybody know something about the AniEditScreen?

### ISSUE THREAD

**Issue by JRF-github** [ja2-stracciatella/ja2-stracciatella#1298]
[//]: # (Thank you for taking the time to report an issue or propose an enhancement. Questions are also welcome.)
[//]: # (If you have more than one item to report, please open separate issues!)

[//]: # (This text will not be visible and you can freely delete it. Check the Preview tab for a cleaner look.)

[//]: # (For bugs, please fill in this table [otherwise delete it]. The lines don't have to stay aligned.)

[//]: # (Please also include:)
[//]: # (- how to reproduce the bug and, if possible, a save game)
[//]: # (- the operating system)
[//]: # (- in case you know, for enhancements, if this feature was already added by some other source mod like 1.13)

[//]: # (When pasting backtraces or any other logs with hashed enumerations, please format it as a code block.)
[//]: # (You can do that by selecting it and clicking the "<>" icon or manually putting three backticks [```] at each end.)
Is there a way to enter this screen other than setting guiCurrentScreen to 6 in the debugger? Also, it requires a "anitest.dat" file, which I can't locate. Does anybody here have it?

**Comment by ja2-derek** [comment]
I don't know anything about the screen, but there is an `anitest.dat` inside `Data/DATA.slf` of JA2 data files.

**Comment by JRF-github** [comment]
Ah, thank you, so the file has just four lines of text. Got it working, it's kind of cute until it crashes.

**Comment by lynxlynxlynx** [comment]
So, is it anything useful to try to bring back to life or should we just cut it out?

**Comment by JRF-github** [comment]
Doesn't look that useful to me, but feel free to judge for yourself. I've pushed my changes [here](https://github.com/JRF-github/ja2-stracciatella/tree/AniViewScreenPlayground).

Press F9 in tactical to enter this screen, then press space to cycle the active soldier through his animations, or press L and then PgUp / PgDown to cycle through a predefined list of anim states that are intended to be loaded from anitest.dat. I've hardcoded the string in this patch because I didn't want to spend my time figuring out where Fs_read is searching for that file.
